server.js

```diff
@@ -36,15 +54,6 @@ const copyValidTables = (oldData, newData) => {
 Promise.resolve(getAllData()).then(allData => {
   let data = allData.airtableData;
 
-  setInterval(function() {
```

Comment:
Can we still have this behaviour on staging? That way when we have a permanent staging environment it will be less likely to be out of sync with airtable.

Reply:
lol I just deleted that code a few minutes before your comment